ts/a11y/explorer.ts

```diff
@@ -240,17 +240,22 @@ export function ExplorerMathDocumentMixin<B extends MathDocumentConstructor<HTML
      */
     public static OPTIONS: OptionList = {
       ...BaseDocument.OPTIONS,
-      enrichSpeech: 'shallow',                   // overrides option in EnrichedMathDocument
       enableExplorer: true,
       renderActions: expandable({
         ...BaseDocument.OPTIONS.renderActions,
         explorable: [STATE.EXPLORER]
       }),
+      sre: expandable({
+        speech: 'shallow',                 // overrides option in EnrichedMathDocument
+        domain: 'mathspeak',               // speech rules domain
+        style: 'default',                  // speech rules style
+        locale: 'en'                       // switch the locale
+      }),
```

Comment:
Since the base document will be an `EnrichDocument`, you should be able to do

``` javascript
      sre: expandable({
        ...BaseDocument.OPTIONS.sre,
        speech: 'shallow'
      }),
```

in order to avoid having to repeat all the options that stay the same.  That way, if any are ever added to the semantic-enrich extension, they will be included here automatically.

Reply:
Done.